jobserver/models.py

```diff
@@ -110,13 +108,6 @@ def runtime(self):
         return Runtime(int(hours), int(minutes), int(seconds))
 
     def save(self, *args, **kwargs):
-        if self.started and not self.started_at:
-            self.started_at = datetime.datetime.now(tz=pytz.UTC)
-
-        if self.status_code is not None and not self.completed_at:
-            if self.started:
-                self.completed_at = datetime.datetime.now(tz=pytz.UTC)
-
```

Comment:
Any reason to not just remove this method altogether and let the superclass handle it?

Reply:
There's still a call to callback_url below the `super()` call, removed in #264